PascalsTriangle.java

```diff
@@ -0,0 +1,44 @@
+// Time Complexity :O(n^2)
+// Space Complexity :O(1)
+// Did this code successfully run on Leetcode :Yes
+// Any problem you faced while coding this :No , just did not get enogh time to code this out in mock interview
+
+
+// Your code here along with comments explaining your approach
+// Every list will have 1 at the start and 1 in the end. The rest of the elements would just be addition of current element and next element starting from 
+// index 0 upto element just before the last element.
+class Solution {
```

Comment:
Good Job! Please add the other problem as well if you have not already added it.